.travis.yml

```diff
@@ -1,31 +1,33 @@
 language: ruby
+sudo: false
 
 rvm:
-  - 2.0.0
-  - 1.9.3
+  - 2.3.1
 
 gemfile:
   - Gemfile
   - Gemfile_ar30
   - Gemfile_ar31
   - Gemfile_ar32
-  - Gemfile_ar40threadsafe
-  - Gemfile_ar42threadsafe
-
-sudo: false
-
-before_script: echo $OLD_RAKE
+  - Gemfile_ar40
```

Comment:
Sorry, I have very little experience with Travis. `Gemfile_ar_master` is supposed to be allowed to fail. Isn't adding it here makes Travis to test it without allowing to fail?